contents/handbook/team-structure.md

```diff
@@ -1,35 +1,108 @@
 ---
-title: Team structure
+title: How small teams work
 sidebar: Handbook
 showTitle: true
 hideAnchor: false
 ---
 
-## Small teams
+We're organized the company into small teams that are multi-disciplinary and as self-sufficient as possible. [You can read about why we've done it this way](/handbook/company/small-teams), or [browse the small teams](/teams).
 
-We've organized the company into small teams that are multi-disciplinary and as self-sufficient as possible. [You can read about why we've done it this way](/handbook/company/small-teams), or [browse the small teams](/teams).
+## Forming new small teams
 
-## Org chart and reporting lines
+Organizational changes usually happen at the small team level. Here’s what happens when we form a new small team: 
 
-We maintain our full org chart in Deel, [which you can access here](https://app.deel.com/organization-chart/organization/834ac289-7c04-4d93-91f0-8922c5664b77?groupBy=group-by-report).
+- [ ] Exec discusses with relevant team leads (and/or managers if applicable).
+- [ ] Exec discusses with relevant team member(s). 
+- [ ] Ops team updates the [Org Chart](https://app.deel.com/organization-chart/organization/834ac289-7c04-4d93-91f0-8922c5664b77?groupBy=group-by-report) in Deel.
+- [ ] Exec informs everyone else in the company in the next all hands session.
 
-Team leads do not necessarily = managers - read more about how we think about management [here](/handbook/company/management). 
+### Team Lead checklist
+Once the team is announced, team leads are responsible for setting the teams up for success. Here's a checklist.
 
-## Organization changes
+- [ ] Team lead adds/removes from relevant [Sentry teams](https://sentry.io/settings/posthog/teams/).
+- [ ] Team lead adds/removes from relevant [GitHub teams](https://github.com/orgs/PostHog/teams).
+- [ ] Team lead adds/removes from Slack `@` groups / team mentions (e.g. `@core-experience`).
+- [ ] Team lead creates a new [teams page and adds content](/teams), team members to it
+- [ ] Team lead creates 1:1s with new team members.
+- [ ] Team lead creates stand-ups, planning sessions, and any other team rituals, including sprint templates.
+- [ ] Team lead creates a new team Slack channel, with descriptions and group handle
+- [ ] Team lead opens a PR on `posthog.com` documenting the change in the handbook.
```

Comment:
IMO this should be removed - the working document is the github issue that is created and we shouldn't have to update two things when this one won't actually be used.